haystack/nodes/answer_generator/openai.py

```diff
@@ -191,7 +209,7 @@ def _build_prompt(self, query: str, documents: List[Document]) -> Tuple[str, Lis
             )
 
         # Top ranked documents should go at the end
-        context = " ".join(reversed(input_docs_content))
+        context = "\n".join(reversed(input_docs_content))
```

Comment:
We use `" "` here as this is what's used in the transition guide for transition from answers endpoint to completions endpoint (see [here](https://help.openai.com/en/articles/6233728-answers-transition-guide#h_89196129b2) for the transition guide and [here](https://github.com/openai/openai-cookbook/blob/5da9f28eb83702b385b1776bbc11b5f1e64f640b/transition_guides_for_deprecated_API_endpoints/answers_functionality_example.py#L254) for the place where `" "` is used). 

What was your motivation for changing this to `"\n"`? Anyway, I assume that this change won't affect the the way the answers are produced, as tokenization should treat both `" "` and `"\n"` the same way.

Reply:
My motivation was to make it more readable. With the newline you can see very quickly the different articles in the context.